packages/components/src/DatePicker/_docs/DatePicker.stories.tsx

```diff
@@ -249,3 +249,73 @@ export const Validation: Story = {
 export const DisabledDays: Story = {
   parameters: { controls: { include: /^disabled/ } },
 }
+
+export const LimitedWindowWidth: Story = {
+  name: "At 400% window size",
+  parameters: {
+    controls: { disable: true },
+    viewport: {
+      viewports: {
+        ViewportAt400: {
+          name: "Viewport at 400%",
+          styles: {
+            width: "320px",
+            height: "350px",
+          },
+        },
+      },
+      defaultViewport: "ViewportAt400",
+    },
+  },
+}
+
+export const LimitedViewportHeight: Story = {
+  name: "Limited viewport height",
+  args: {
+    labelText: "Calendar with reduced space below",
+  },
+  decorators: [
+    Story => (
+      <div className="mb-[150px]">
+        <Story />
+      </div>
+    ),
```

Comment:
Is it possible to one-line the decorators?

```suggestion
    Story => <div className="mb-[150px]"><Story /></div>,
```

Reply:
Sadly it lints to a multi-line